app/notification.go

```diff
@@ -174,7 +175,7 @@ func (a *App) SendNotifications(post *model.Post, team *model.Team, channel *mod
 			}
 			umc <- nil
 		}(id)
-		updateMentionChans = append(updateMentionChans, umc)
+		mentionAutofollowChans = append(mentionAutofollowChans, umc)
```

Comment:
Probably the channel variable shoud be `mac` here too, instead of reuse the `umc` name.